proto/viam/app/v1/app.proto

```diff
@@ -269,11 +275,13 @@ message LogEntry {
 
 message GetRobotPartLogsResponse {
   repeated LogEntry logs = 1;
+  optional PageToken next_page_token = 5;
```

Comment:
PageToken -> string

Reply:
And remove the optional.

Reply:
need the optional to not break the frontend for backwards compatibility -- ran into this yesterday with the filter

Reply:
hmm. Do we construct this on the FE anywhere. It should only complain if we are creating a object and don't supply all the fields. In this case it shouldn't matter.

We need to be able to support adding new fields without optional or it's going to make our apis really cumbersome to use. proto3 allows new fields to be added without marking them as optional.

Reply:
Hmm actually maybe it's something that is bound to Request objects with the linter. It would make sense that the linter doesn't complain to me for the response without explicating setting optional but let me try locally.

Reply:
yup this is fine w/o the optional here too